src/test/java/org/mybatis/spring/annotation/MapperScanTest.java

```diff
@@ -90,6 +91,19 @@ void assertNoMapperClass() {
     }
   }
 
```

Comment:
Please add a test for repeatable annotation.

Reply:
Thank you for the feedback. I added tests for repeating `MapperScan` pattern and `MapperScans` pattern.